.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+Status: **Ready for Review** or **Open for Visibility**
+Reviewers: GitHub @<name>s of all reviewers
+
+Linked PRs: <links to corresponding PRs, optional>
+
+## Changes
+- <changes here>
+
+## Todos:
+- [ ] +1
+- [ ] Updated README
+- [ ] Updated CHANGELOG
+- [ ] <Other applicable TODOs>
+
+### Feedback:
```

Comment:
How much is this used in PRs on teams? On ThinkGeek I'm not sure I ever saw feedback added here.